src/fabric/src/fabric2_active_tasks.erl

```diff
@@ -0,0 +1,35 @@
+-module(fabric2_active_tasks).
+
+
+-export([
+    get_active_tasks/0,
+    get_active_task_info/1,
+
+    update_active_task_info/2
```

Comment:
Check the file with emilio, it might complain that the order of exports doesn't match the definition order. So might have to put `update_active_task_info/2` first, or change the definition order, whichever looks better.